simplemind/models.py

```
@@ -79,7 +53,7 @@ class Conversation(SMBaseModel):
     def __str__(self):
         return f"<Conversation id={self.id!r}>"
 
-    def add_message(self, role: str, text: str, meta: Dict[str, Any] = {}):
+    def add_message(self, role: MESSAGE_ROLE, text: str, meta: Dict[str, Any] = {}):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)